features/step_definitions/attachment_steps.rb

```diff
@@ -49,7 +49,13 @@ def attachment_path(filename)
 
 Then /^the attachment should have the same content type as the fixture "([^"]*)"$/ do |filename|
   in_current_dir do
-    require 'mime/types'
+    begin
+      # Use mime/types/columnar if available, for reduced memory usage
+      require 'mime/types/columnar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.